docset/winserver2025-ps/smbshare/Get-SmbClientAccessToServer.md

```diff
@@ -0,0 +1,199 @@
+---
+description: Use this topic to help manage Windows and Windows Server technologies with Windows PowerShell.
+external help file: SmbClientAccessToServer.cdxml-help.xml
+Module Name: SmbShare
+ms.date: 02/22/2024
+online version: https://learn.microsoft.com/powershell/module/smbshare/get-smbclientaccesstoserver?view=windowsserver2025-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Get-SmbClientAccessToServer
+---
+
+# Get-SmbClientAccessToServer
+
+## SYNOPSIS
+Gets information about SMB client access to a specified server.
+
+## SYNTAX
+
+### Query
+
+```
+Get-SmbClientAccessToServer [-Name] <String[]> [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>]
+ [-AsJob] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### InputObject
+
+```
+Get-SmbClientAccessToServer -InputObject <CimInstance[]> [-CimSession <CimSession[]>]
+ [-ThrottleLimit <Int32>] [-AsJob] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Get-SmbClientAccessToServer` cmdlet is used to retrieve information about SMB client access to
+a specified server. When you run this cmdlet, it'll return a list of all SMB client connections to
```

Comment:
not a list of connections, but a list of allowed and blocked connections for SMB over QUIC client access control